src/components/CheckableInput/CheckableInput.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from "classnames";
+import { TSFixMe } from "index";
+import { nanoid } from "nanoid";
+import PropTypes from "prop-types";
+import React, { useEffect, useRef, HTMLProps } from "react";
+
+type Props = {
+  type: string;
```

Comment:
Can you define the valid types here 
```suggestion
  type: "checkbox" | "radio";
```